main.go

```diff
@@ -86,17 +91,21 @@ func showTimeDuration(durations map[string]time.Duration) {
 }
 
 // showErrorMessages shows table with all error messages gathered during action
-func showErrorMessages(action string, errorMessages map[string]error) error {
-	if len(errorMessages) > 0 {
+func showErrorMessages(action string, errorMessages map[string]LogMessage) error {
+	if hasPriorityErrors(errorMessages, log.CurrentLevel()) {
 		fmt.Println()
 		fmt.Printf("The following errors were encountered during %s:\n\n", action)
 		w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
-		_, _ = fmt.Fprintln(w, "STEP\tERROR\t")
-		for svc, err := range errorMessages {
-			_, _ = fmt.Fprintf(w, "%v\t%v\n", svc, err)
+		_, _ = fmt.Fprintln(w, "TYPE\tSTEP\tERROR\t")
+		for step, er := range errorMessages {
```

Comment:
Why `er` and not `err`? The `err` is convention. I think.

Reply:
Exactly because of that, I wasn't sure to use err that is a convention that represents an error, to represent the LogMessage struct.